php/values.yaml

```diff
@@ -377,6 +377,105 @@ fpm:
+      ; CGI/FastCGI.
+      ; http://php.net/ini.sections
+
+      ; Directives are specified using the following syntax
+      ; directive = value
+      ; Directive names are *case sensitive* - foo=bar is different from FOO=bar.
+      ; Directives are variables used to configure PHP or PHP extensions.
+      ; There is no name validation.  If PHP can't find an expected
+      ; directive because it is not set or is mistyped, a default value will be used.
+
+      ; The value can be a string, a number, a PHP constant (e.g. E_ALL or M_PI), one
+      ; of the INI constants (On, Off, True, False, Yes, No and None) or an expression
+      ; (e.g. E_ALL & ~E_NOTICE), a quoted string ("bar"), or a reference to a
+      ; previously set variable or directive (e.g. ${foo})
+
+      ; Expressions in the INI file are limited to bitwise operators and parentheses
+      ; |  bitwise OR
+      ; ^  bitwise XOR
+      ; &  bitwise AND
+      ; ~  bitwise NOT
+      ; !  boolean NOT
+
+      ; Boolean flags can be turned on using the values 1, On, True or Yes.
+      ; They can be turned off using the values 0, Off, False or No.
+
+      ; An empty string can be denoted by simply not writing anything after the equal
+      ; sign, or by using the None keyword
+
+      ;  foo =         ; sets foo to an empty string
+      ;  foo = None    ; sets foo to an empty string
+      ;  foo = "None"  ; sets foo to the string 'None'
+
+      ; If you use constants in your value, and these constants belong to a
+      ; dynamically loaded extension (either a PHP extension or a Zend extension),
+      ; you may only use these constants *after* the line that loads the extension.
+
+      ;;;;;;;;;;;;;;;;;;;
+      ; About this file ;
+      ;;;;;;;;;;;;;;;;;;;
+      ; PHP comes packaged with two INI files. One that is recommended to be used
+      ; in production environments and one that is recommended to be used in
+      ; development environments.
+
+      ; php.ini-production contains settings which hold security, performance and
+      ; best practices at its core. But please be aware, these settings may break
+      ; compatibility with older or less security conscience applications. We
+      ; recommending using the production ini in production and testing environments.
+
+      ; php.ini-development is very similar to its production variant, except it is
+      ; much more verbose when it comes to errors. We recommend using the
+      ; development version only in development environments, as errors shown to
+      ; application users can inadvertently leak otherwise secure information.
+
+      ; This is php.ini-production INI file.
+
+      {{- range $k, $v := .Values.fpm.ini }}
+      {{- if eq (printf "%t" $v) "true" }}
+      {{ snakecase $k }} = On
+      {{- else if eq (printf "%t" $v) "false" }}
+      {{ snakecase $k }} = Off
+      {{- else if eq (printf "%T" $v) "string" }}
+      {{ snakecase $k }} = "{{ $v }}"
```

Comment:
@cw-niinuma 
I don't think it works because `errorReporting: E_ALL & ~E_NOTICE` is changed to `error_reporting = "E_ALL & ~E_NOTICE"`.

I don't have a good idea.
```
{{- if eq $k "errorReporting" }}
{{ snakecase $k }} = {{ $v }}
{{- end }}
```
This is a bad idea. But what about this approach if you have no other ideas?

Reply:
@cw-ozaki  I'll try to think about it.